examples/util/base.js

```diff
@@ -137,11 +138,11 @@ const mobilenet_v2_onnx = {
     softmax: true,
   },
   intro: 'MobileNetV2 improves the state of the art performance of mobile models.',
-  paperUrl: 'https://arxiv.org/abs/1801.04381',
-  modelCDNUrl: 'https://cdn_host_and_path/mobilenetv2-1.0.onnx'
+  paperUrl: 'https://arxiv.org/abs/1801.04381'
 };
 const resnet_v1_onnx = {
   modelName: 'resnet_v1_onnx',
+  modelSize: '46.8MB',
```

Comment:
the size of resnet50 v1 is 102.6MB. This is the size of resnet18 v1.

Reply:
I have once used the resnet18 but changed in https://github.com/intel/webml-polyfill/pull/353. You might need to update the local models